src/hotspot/cpu/aarch64/c1_LIRGenerator_aarch64.cpp

```diff
@@ -426,7 +426,7 @@ void LIRGenerator::do_ArithmeticOp_FPU(ArithmeticOp* x) {
     tmp = new_register(T_DOUBLE);
   }
 
-  arithmetic_op_fpu(x->op(), reg, left.result(), right.result(), NULL);
+  arithmetic_op_fpu(x->op(), reg, left.result(), right.result(), x->is_strictfp());
```

Comment:
Not a question on this change, but a broader one raised by this and the previous change.  The changes you're making here and in the previous file appear in the Linux version in the arm directory, with the ones in the previous file in an "#ifdef AARCH64" block.  Does that mean that on Linux, the aarch64 directory is no longer used, but rather arm32 and arm64 are both in the arm directory?  Similar to x86 and x86_64 both being in x86, powerpc and powerpc64 both using powerpc, etc.  I'd like to understand the relationship between the code in arm and aarch64.  In particular which files get used when compiling on a Linux arm64 system?  The equivalent ones to these or the ones in arm or both?

This might just be my unfamiliarity with this particular architecture.